src/Language/ASKEE/Exposure/Syntax.hs

```diff
@@ -154,6 +155,7 @@ prefixFunctionName ident =
     "table"       -> Right FTable
     "simplify"    -> Right FSimplify
     "withParams"  -> Right FWithParams
+    "modelSkillRank" -> Right FSkillRank
```

Comment:
I'm admittedly far from an expert on the subject, but I wonder about the name of this function. IIUC, none of the arguments to `modelSkillRank` are actually models, right?

Reply:
While I think they _could_ be in a more sophisticated version (e.g. one that performs sampling automatically), you're right that the input is really an (array of an) array of samples.